### ISSUE TITLE
That filter feature is a great idea, and I think it could be more

### ISSUE THREAD

**Issue by CactusMikaz** [AlonsoAliaga/AlonsoTags#5]
NOTICE THAT THIS IS JUST A REQUEST. FEATURE CAN NEVER BE ADDED.

**Is your feature request related to a problem? Please describe.**
You make a nice filter to let player could view all/unlock tags in GUI, that is awesome.

I was thinking may be I could add my own filter, it's like a group of these tags.
I will write down some example in below sections.

**Describe the solution you'd like**
All/unlock filter is great, maybe add another "Locked" filter to let players know how many tags he haven't obtain is a good Idea.
These three "All/unlock/locked" will be a default filter.

Maybe I could add some tags into specific group, and make it a filter that player could switch.
I could set tags called "Tiger" or "Monkey", and these tags are both in groups called "Zoo's Tags" and "VIP's Tags", so you can find these tags when filter switch to "Zoo's Tags" or "VIP's Tags".

Actually I developed a plugin that provide player custom hats, also using these filter design to let player could quickly locate hats he want to finds in over 50 hats.

![image](https://user-images.githubusercontent.com/64403069/131543896-44f57f15-a68c-4720-a0ee-e4e6c6790650.png)
This is filters or groups of hats, green wool is player's unlock hats, red wool is player's locked hats.

![image](https://user-images.githubusercontent.com/64403069/131544114-28de99c5-4ceb-476f-9e71-a6d3e69e05a0.png)
Inside the desert group( that sand material ), there are five hats in it. Because their config set to desert group.

**Describe alternatives you've considered**

**Additional context**

**Comment by AlonsoAliaga** [comment]
I cant tell you from now that categorirs for tags will never be added. People usually think that itxs just writting some lines and make it work, but thats not true. I had to modify a lot of the current code just to make filtering tags possible, however I have no plans to spend so many hours for categories. It was already decided since I decided to start with this project and that wont change, adding categories is not on my future plans, sorry
